src/pybind/mgr/dashboard/frontend/src/app/shared/enum/retention-frequency.enum.ts

```diff
@@ -11,7 +12,8 @@ export enum RetentionFrequencyCopy {
   h = 'Hourly',
   d = 'Daily',
   w = 'Weekly',
-  m = 'Monthly',
+  M = 'Monthly',
+  m = 'Minutely',
```

Comment:
Is it going to be in this order in the drop down? I see we are following `ascending order` for all items and according to that this should be first item.

Reply:
good catch. done 👍🏽